src/content/patterns/adding-creating-pattern/adding-creating-pattern.mdx

```diff
@@ -0,0 +1,153 @@
+
+The creation of resources can vary considerably in complexity and with this in mind there are a number of alternatives to consider when designing a creation flow.
+
+<AnchorLinks small>
+
+- [Advanced Option](#advanced-option)
+- [Stepped creation](#stepped-creation)
+- [Illustration or Image](#illustration-or-image)
+- [Single scrollable page](#single-scrollable-page)
+- [Button position](#button-position)
+
+</AnchorLinks>
+
+<ImageComponent cols="8">
+
+![Example](images/2.jpg)
+
+</ImageComponent>
+
+
+### Advanced Option
+
+Consider having an advanced option to avoid overwhelming users, while still allowing more advanced users to access everything.
+
+In situations where there are large numbers of options consider, by default, having only a minimal set visible. Provide the ability for users to chose if they want to expose more advanced, optional, options. Users should always be able to get back out of advanced.
+
+It is recommended that if the user switches to the advanced mode this is remembered and the next time they enter the same creation process it starts in advanced .
+
+<ImageComponent cols="4">
+
+![Example](images/3.jpg)
+
+</ImageComponent>
+<br />
+<ImageComponent cols="8">
+
+![Example](images/4.jpg)
+
+</ImageComponent>
+
+Take into consideration that even if by default there are only a few options visible, you might want the ability to expose additional advanced options meaning more space would be required.
+
+
+### Stepped creation
+Consider whether breaking the process up into steps could help the user. 
+If the creation task is complex or involves a large number of mandatory options, think about:
+- Displaying related options together
+- Grouping dependent options together
+- Using the space to provide additional information / explanatory text for the option(s) 
+- Using this as a simple stepped version with minimal options and extra guidance alongside the advanced option to expose all options 
+
+<ImageComponent cols="8">
+
+![Example](images/5.jpg)
+
+</ImageComponent>
+
+
+### Illustration or Image
+Consider if an illustration or image would help your users.
+
+If you think an illustration or image could help explain the creation flow or task to your user or help with clarity then it should be placed in the right hand side of the creation panel.
+
+<ImageComponent cols="8">
+
+![Example](images/6.jpg)
+
+</ImageComponent>
+
+### Single scrollable page
+Having just a single scrollable page of options is not desirable however sometimes it might be necessary.  For example in the situations when the advanced option is used it will likely be a long scrollable list of options.
+
+In this situation it is recommended that the buttons should be located in the top right and should stick upon scrolling, meaning they are always visible to the user and they don’t have to scroll to the bottom of the page to find them.
```

Comment:
I think sticking buttons to the bottom of a page is acceptable. I don't think the only place we'd recommend having them is on the side. I agree with that they should always be visible on the page.